src/pages/access_review/AppSummary.js

```diff
@@ -108,7 +97,7 @@ export const AppSummary = hh(class AppSummary extends React.Component {
             },
             [
               StructuredDarRp({darInfo: darInfo}),
-              div({},[
+              div({isRendered: !isNil(mrDAR)},[
```

Comment:
DARs with unreviewed, canceled, closed election statuses return a 404 when findElectionById is called (in ReviewResults line 63). So the access Election is null, and in AppSummary the accessElection is only used for the  mrDAR download link, so just remove that link if it is null. Similarly, in DarApplication VoteSummaries are not rendered for the DARs with null accessElection.